virtool/uploads/models.py

```diff
@@ -0,0 +1,24 @@
+from sqlalchemy import Column, String, Boolean, Integer, DateTime
+
+from virtool.postgres import Base
+
+
+class Upload(Base):
+    __tablename__ = "uploads"
+
+    id = Column(Integer, primary_key=True)
+    created_at = Column(DateTime)
+    field = Column(String)
+    name = Column(String, unique=True)
+    name_on_disk = Column(String, unique=True)
+    ready = Column(Boolean)
+    removed = Column(Boolean)
+    reserved = Column(Boolean)
+    size = Column(Integer)
+    type = Column(String)
```

Comment:
Please look into [enumerated types](https://www.postgresql.org/docs/13/datatype-enum.html) for this.

Possible values are:
- reference
- reads
- subtraction
- `null`

For `null`, I am not sure if this has to be in the enum.

When `type` is `null` it means that the file type is not explicitly understood by Virtool and does not map to a specific resources type (eg. reads -> sample, subtraction -> subtraction).